test/custom_test_data/member_row.yml

```diff
@@ -0,0 +1,9 @@
+:url: 'http://www.parliament.go.tz/mps-list'
```

Comment:
You've a slightly odd mismatch going on by providing a URL here, as we normally use that for orchestrating the test. But yet you're not supplying the classname, but instead supplying that in the file, along with newly hard-coded info (i.e. that you're actually testing against `member_rows.first` person in the list).

I think it would be useful if `scraper_test` were extended to allow you to specify the filtering you want (in this case `'id': '14'`) and so allow tests like this to run without needing an explicit `test.rb` file.

Can you raise an issue for that, quoting this, and anywhere else you've done similar, as examples?

Then I suggest you treat this as a step towards it, hoist the class name to here as per the normal `scraper_test` approach, and then hoist the other filtering options as well.

If you think that's too much work or too difficult, then I suggest you simply raise the issue, and then remove the URL from here, so it's not in quite so much of an uncanny valley.